basic/github_dynamic_targets/github_dynamic_targets.tf

```diff
@@ -1,56 +1,12 @@
-provider "sym" {
-  org = "sym-example"
-}
-
-provider "aws" {
-  region = "us-east-1"
-}
-
-############ General AWS Secrets Manager Setup ##############
-
-# Creates an AWS IAM Role that the Sym Runtime can use for execution
-# Allow the runtime to assume roles in the /sym/ path in your AWS Account
-module "runtime_connector" {
-  source  = "symopsio/runtime-connector/aws"
-  version = ">= 1.0.0"
-
-  # The aws/secretsmgr addon is required to access secrets
-  addons = ["aws/secretsmgr"]
-
-  environment = "main"
-}
-
-# An Integration that tells the Sym Runtime resource which AWS Role to assume
-# (The AWS Role created by the runtime_connector module)
-resource "sym_integration" "runtime_context" {
-  type = "permission_context"
-  name = "main-runtime"
-
-  external_id = module.runtime_connector.settings.account_id
-  settings    = module.runtime_connector.settings
-}
-
-# This resource tells Sym which role to use to access your AWS Secrets Manager
-resource "sym_secrets" "this" {
-  type = "aws_secrets_manager"
-  name = "main-sym-secrets"
-
-  settings = {
-    context_id = sym_integration.runtime_context.id
-  }
-}
-
-############ GitHub Integration and Secret Setup ##############
```

Comment:
Do we want to keep this comment?

Reply:
Oh I did keep it. It's at the top of `github_dynamic_targets.tf`.

Reply:
Unless you mean the `############ GitHub Integration and Secret Setup ##############`?

Reply:
We would have to add a similar header for every file if we do that so decided against it! Same decision applies for similar comments below.